### PR TITLE
Build overview header

### PR DESCRIPTION
**Pull request by djones**

### CODE REVIEW

File: tests/acceptance/build-test.js [hunk 1]
```diff
@@ -137,8 +137,8 @@ describe('Acceptance: Build', function() {
     });
     percySnapshot(this.test.fullTitle() + ' on the build page');
 
-    click('.BuildModePicker button');
-    percySnapshot(this.test.fullTitle() + ' in overview mode');
+    //click('.BuildModePicker button');
```
Comment (djones) [inline tests/acceptance/build-test.js]: these commented lines are removed in another PR.

File: tests/integration/components/build-overview-test.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+/* jshint expr:true */
```
Comment (fotinakis) [inline tests/integration/components/build-overview-test.js]: :man_dancing: 🍰

**Comment by fotinakis** [comment]
One more comment: I think we should say "in 15 snapshots" rather than "from 15 snapshots"

File: app/styles/components/_build_overview.sass [hunk 3]
```diff
@@ -0,0 +1,58 @@
+$statusBarWidth: 10px
```
Comment (djones) [inline app/styles/components/_build_overview.sass]: Good catch. Interestingly this approach fixes the global scope issue:

```
.BuildOverview
  $statusBarWidth: 10px
  width: $statusBarWidth // OK to use it here

body
  width: $statusBarWidth // Undefined variable error here.
```

Comment (djones) [inline app/styles/components/_build_overview.sass]: Fixed with 8df73b2b70e95a7ad4226e7311a2cacf05e2d925

**Pull request merged**